modules/programs/pls.nix

```
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.pls;
+
+  aliases = {
+    ls = "${pkgs.pls}/bin/pls";
+    ll =
+      "${pkgs.pls}/bin/pls -d perms -d user -d group -d size -d mtime -d git";
+  };
+
+in {
+  meta.maintainers = [ hm.maintainers.kalhauge ];
+
+  options.programs.pls = {
+    enable =
+      mkEnableOption "pls, a modern replacement for <command>ls</command>";
+
+    enableAliases = mkEnableOption "recommended pls aliases";
+  };
+
```

It would be useful to have a package option.
```suggestion
  options.programs.pls = {
    enable =
      mkEnableOption "pls, a modern replacement for <command>ls</command>";
      
    package = mkPackageOption pkgs "pls" { };

    enableAliases = mkEnableOption "recommended pls aliases";
  };

```

---

Okay, I thought I heard that package options were only recommended when there actually were multiple packages for an application. No problem though, I added it now.